wolfssl/internal.h

```diff
@@ -146,14 +146,14 @@
         /* do nothing */
 #elif defined(WOLFSSL_CMSIS_RTOS)
     #include "cmsis_os.h"
+#elif defined(WOLFSSL_CMSIS_RTOSv2)
+    #include "cmsis_os2.h"
+#elif defined(WOLFSSL_CMSIS_RTOSv2)
+    #include "cmsis_os2.h"
 #elif defined(WOLFSSL_MDK_ARM)
-    #if defined(WOLFSSL_MDK5)
```

Comment:
Will removing this include for "cmsis_os.h" break existing users builds?  In other words, were previous users defining WOLFSSL_MDK_ARM and WOLFSSL_MDK5 without defining WOLFSSL_CMSIS_RTOS?

Reply:
Rollbacked